language-server-commons/src/test/java/com/palantir/ls/services/DefaultTextDocumentServiceTest.java

```diff
@@ -77,8 +78,7 @@
 import org.mockito.Mock;
 import org.mockito.MockitoAnnotations;
 
-// TODO (darora): #131 split this up and test things independently in the language-server-commons as well
-public final class DefaultTextDocumentServiceTest {
+public class DefaultTextDocumentServiceTest {
```

Comment:
lol I'm guessing now that you did a find and replace all instances of "public final class"...